frontend/src/queries/nodes/DataTable/ExportWithConfirmation.tsx

```diff
@@ -18,6 +18,7 @@ export function ExportWithConfirmation({
 }: ExportWithConfirmationProps): JSX.Element {
     return (
         <Popconfirm
+            zIndex={10000}
```

Comment:
This is likely going to mess up the ordering of other things.
We prefer to use z-indexes as a CSS variables and have a list of them [here](https://github.com/PostHog/posthog/blob/dfd76f4c71103e079992206df3e62c87c6b9bdcf/frontend/src/styles/vars.scss#L251-L274).